docs/en/stack/security/authorization/privileges.asciidoc

```diff
@@ -29,6 +29,9 @@ This includes snapshotting, updating settings, and rerouting. It also includes
 obtaining snapshot and restore status. This privilege does not include the 
 ability to manage security.
 
+`manage_ccr`::
+TBD
```

Comment:
Roughly: This is cluster privilege needed on a cluster holding following indices to grant permissions to manage following indices (e.g., pause, and resume) and manage auto-follow patterns (@martijnvg double check me here please).